object-construction-checker/tests/file/ACFileTest.java

```diff
@@ -0,0 +1,5 @@
+
+
+public class FileACTest {
```

Comment:
Can we keep all the file- and socket-related changes out of this PR?  This includes the size 0 `.astub` files; not sure how they even got added here.

Reply:
Sorry! I added this by mistake!